data/campaigns/Northern_Rebirth/scenarios/02_01_Infested_Caves.cfg

```diff
@@ -22,21 +22,21 @@
             story= _ "Grief for the fallen could not suppress the people’s joy in their newfound freedom. They ransacked the orcish storehouses, sang, danced and drank with abandon. New-fledged warriors boasted of their deeds in the battle, while the women and children looked on with awe."
         [/part]
         [part]
-            story= _ "Amidst all the revelry, Tallin alone remained grave. He knew it likely that the orcs would return with a vengeance and slaughter every last one of them. The following morning, he gathered the cooler heads about him so they could set their next course. How might they, untrained in the art of war and without expectation of aid from the prosperous human lands of the south, defend themselves?"
+            story= _ "Amidst all the revelry, Tallin alone remained grave. He knew it likely that the orcs would return with a vengeance and slaughter every last one of them. The following morning, he gathered the cooler heads so they could decide their next course. How might they, untrained in the art of war and without expectation of aid from the prosperous human lands of the south, defend themselves?"
         [/part]
         [part]
-            story= _ "The war council’s thoughts quickly turned to the dwarves, close allies in the past. However, nobody knew the state of things in the Knalgan tunnels. The bravest of scouts found in there the spoor of many trolls, and of creatures darker than trolls. It was even rumored that the dwarven dead from the invasion of Knalga had risen in unlife, roaming the tunnels to slay the living. Fighting through all manner of unknown monsters in the mere hope of finding surviving dwarves held but little appeal."
+            story= _ "The war council’s thoughts quickly turned to the dwarves, close allies in the past. However, nobody knew the state of things in the Knalgan tunnels. The bravest of scouts found in there the spoor of many trolls, and of even viler creatures. For years rumors had spread of the dwarves, slain in the invasion of Knalga, now risen in unlife to roam the tunnels. To Tallin and the others, fighting through all manner of unknown monsters in the mere hope of finding surviving dwarves held but little appeal."
```

Comment:
The "For years rumors had spread ..." scans wrongly to me. In the original, the subject is "dwarven dead from the invasion of Knalga" and the rumor is that they "had risen in unlife, ...". In the new one, the subject seems to be the whole of "dwarves, slain in the invasion of Knalga, now risen in unlife to roam the tunnels" and there isn't a rumor, just a subject.

Reply:
I added the rumor part because they would have no idea that undead were in the caves.

Reply:
I made the first change. However, Unlife is how this campaign talks about undead.I figured I'd leave it the way it is.